k8s/deploy/blueprint.yaml

```diff
@@ -0,0 +1,50 @@
+# The information below is boilerplate for the blueprint.
+apiVersion: xl/v1
+kind: Blueprint
+
+# This metadata helps the next user better understand what the blueprint is about.
+
+metadata:
+  projectName: Kubernetes YAML
+  description: |
+    The blueprint creates an application package with XL YAML files and Kubernetes YAML
+  author: XebiaLabs
+  version: 1.0
+
+parameters:
```

Comment:
Can you move the `parameters` section underneath a new `spec` section? It's a change we've made to make the blueprints YAML file conform to K8s conventions (see [LOVE-401](https://xebialabs.atlassian.net/browse/LOVE-401) and [LOVE-415](https://xebialabs.atlassian.net/browse/LOVE-415)).

If you want to test this, please run the blueprint with the latest `master` build of the [xl-cli](https://github.com/xebialabs/xl-cli). Or wait until XL CLI 8.6 is released.

Reply:
Okay